### PR TITLE
launcher icon for the lead-management-android

### PR DESCRIPTION
**Pull request by aiyaad**
xxhdpi

xhdpi

hdpi
mdpi

### CODE REVIEW

**Comment by garg-anuj** [comment]
Icon do not need to have JBoss logo in it and rest of content also is not related to app content

**Comment by aiyaad** [comment]
@garg-anuj i have removed the JBoss logo, and have reconstructed the launcher icon.

File: Replacing minmap launchers. [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (aiyaad) [inline Replacing minmap launchers.]: xxhdpi

xhdpi

hdpi
mdpi

**Comment by garg-anuj** [comment]
If you haven't already please sumit your task. Work is good to be approved.

**Pull request closed without merging**